validate.go

```
@@ -331,11 +331,12 @@ func validateDiscriminator(r Registry, s *Schema, path *PathBuffer, mode Validat
 		return
 	}
 
-	Validate(r, r.SchemaFromRef(ref), path, mode, v, res)
+	ValidateAndSetDefaults(r, r.SchemaFromRef(ref), path, mode, v, res)
 }
 
 // Validate an input value against a schema, collecting errors in the validation
-// result object. If successful, `res.Errors` will be empty. It is suggested
+// result object. And set default values on ommited fields
```

golangci-lint

`ommited` is a misspelling of `omitted`

(misspell)